dsm/provider.go

```diff
@@ -87,6 +87,7 @@ func Provider() *schema.Provider {
 			"dsm_azure_group":         resourceAzureGroup(),
 			"dsm_secret":              resourceSecret(),
 			"dsm_group":               resourceGroup(),
+			"dsm_existing_group":      resourceExistingGroup(),
```

Comment:
How about doc file for the new resource? Doesn't it require?

Reply:
Can we add this documentation @darktower